src/Workspaces/Core/Portable/Options/GlobalOptionService.cs

```diff
@@ -32,9 +32,11 @@ internal class GlobalOptionService : IGlobalOptionService
 
         private readonly object _gate = new object();
 
+#pragma warning disable IDE0044 // Add readonly modifier - https://github.com/dotnet/roslyn/issues/33009
         private ImmutableDictionary<string, (IOption? option, IEditorConfigStorageLocation2? storageLocation)> _neutralEditorConfigKeysToOptions = s_emptyEditorConfigKeysToOptions;
         private ImmutableDictionary<string, (IOption? option, IEditorConfigStorageLocation2? storageLocation)> _csharpEditorConfigKeysToOptions = s_emptyEditorConfigKeysToOptions;
         private ImmutableDictionary<string, (IOption? option, IEditorConfigStorageLocation2? storageLocation)> _visualBasicEditorConfigKeysToOptions = s_emptyEditorConfigKeysToOptions;
+#pragma warning restore IDE0044 // Add readonly modifier
```

Comment:
why?

Reply:
Sam already fixed this issue in the analyzer - https://github.com/dotnet/roslyn/issues/33009. We should be able to remove this once we move to a newer CodeStyle package which has the fix.